quantum/rgb_matrix_animations/typing_heatmap_anim.h

```diff
@@ -27,6 +31,7 @@ void process_rgb_matrix_typing_heatmap(keyrecord_t* record) {
     }
 }
 
+uint8_t tick = 0;
```

Comment:
This is in a .h file and should be made unique enough for this effect to not interfere with other animations. While tick is not used yet in the rgb matrix, it's too generic of a name. Please change this to something more unique such as heatmap_decrease_timer